### DISCUSSION TITLE
About sponsorship & Insiders

### DISCUSSION THREAD

**Discussion by hellodword** [RaspAP/raspap-webgui#997] [General]
The [sponsor page](https://github.com/sponsors/RaspAP) says: 

> you will be added as a team member to the super-secret private GitHub repository

And the [doc of github organization membership](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/about-organization-membership) says:

> When you accept an invitation to join an organization, the organization owners may be able to see:
> - Your public profile information
> - Your email address
> - If you have two-factor authorization enabled
> - Repositories you have access to within the organization, and your access level
> - Certain activity within the organization
> - Country of request origin
> - Your IP address

No offense, but is there another way to be a sponsor and access to **Insiders** without github team?

**Comment by billz** [comment]
I have no visibility into sponsor details beyond email address, organization repo access and public profile (no IP, country, etc). 

That said, privacy is a valid concern so you may also sponsor directly via Open Collective, our fiscal host.

https://opencollective.com/raspap
https://opencollective.com/privacypolicy
